python/pylibcudf/pylibcudf/tests/test_string_find_multiple.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+
+
+def test_find_multiple():
+    arr = pa.array(["abc", "def"])
+    targets = pa.array(["a", "c", "e"])
+    plc_result = plc.strings.find_multiple.find_multiple(
+        plc.interop.from_arrow(arr),
+        plc.interop.from_arrow(targets),
+    )
+    result = plc.interop.to_arrow(plc_result)
+    expected = pa.chunked_array(
+        [
+            pa.array(
+                [
+                    [elem.find(target) for target in targets.to_pylist()]
+                    for elem in arr.to_pylist()
+                ],
+                type=result.type,
+            )
+        ]
+    )
+    assert result.equals(expected)
```

Comment:
No need to change it in this PR, but in general you don't need to create the chunked array from a normal array manually. If you use the `assert_column_eq` and similar functions from our utils it will handle that for you (or at least it should; if there are cases that aren't handled we should address that). We should toss those kinds of conversions where we don't actually care about the differences into that function so that we don't have to think about it when writing tests.